osf/models/node.py

```diff
@@ -2383,14 +2383,21 @@ def add_contributor(self, *args, **kwargs):
     # Overrides ContributorMixin
     def _add_related_source_tags(self, contributor):
         osf_provider_tag, created = Tag.all_tags.get_or_create(name=OsfSourceTags.Osf.value, system=True)
+
+        if self.is_collected:
+            collection_provider_id = self.collecting_metadata_list[0].collection.provider._id
+            collection_provider_tag, created = Tag.all_tags.get_or_create(name=provider_source_tag(collection_provider_id, 'collections'), system=True)
```

Comment:
nit: Here and elsewhere, we're assigning this 'created' value and never reading it. Do we wind up looking at this value in the debugger, or is it safe to discard into _?

Reply:
For both, it's safe to discard into `_`. I'll change those.